infrastructure/environments/dplplat01/sites.yaml

```diff
@@ -38,6 +38,7 @@ sites:
     description: "Eksempel på bibliotek der kører på 'webmaster' plan, og derfor har et modultest-miljø"
     deploy_key: "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAILj8lXv7C/7s7te9sEpwcHQhgDWfzsCkAN7rqQ4sdTzk"
     plan: webmaster
+    webmaster-cms-version: "2024.27.1"
```

Comment:
The other release related entry name is called `dpl-cms-release`. This is called `webmaster-cms-version` and is used for the `moduletest` environment.

I wonder if we can come up with something that shows a closer relation between the parts. Suggestion:

- `webmaster-dpl-cms-release` - the existing name prefixed with `webmaster` to show the relationship with the plan
- `moduletest-dpl-cms-release`- the existing name prefixed with the environment name
- `webmaster-moduletest-release` - both plan and environment name without the `dpl-cms` name